docs/Configuration.md

```diff
@@ -998,7 +998,7 @@ module.exports = browserResolve.sync;
 By combining `defaultResolver` and `packageFilter` we can implement a `package.json` "pre-processor" that allows us to change how the default resolver will resolve modules. For example, imagine we want to use the field `"module"` if it is present, otherwise fallback to `"main"`:
 
 ```js
-module.exports = (path, options) => {
+module.exports = (request, options) => {
```

Comment:
Good idea. Docs above mention `path`, so I would leave it here as is, but would change `request` below. Just to keep it all in sync.

Reply:
Done 👍